css/main.sass

```diff
@@ -376,11 +376,10 @@ tbody, tr
     display: block
     width: 100%
     margin: 0 auto
-    max-width: 8in
 
 td
+    width: auto //Fixes 3 rows but then creates scaling issue
```

Comment:
If you want to make this responsive then the solution is to stop using the table entirely, not to try to coerce the table into not being a table anymore

Reply:
I think i'll leave the table as a table because i feel it'll be more of a headache then its already been.

Reply:
Actually I think you'll find that you basically don't need any special styling (other than center alignment) if you take the elements out of the table and just make them divs with `display: inline-block`

Reply:
ok ill work on it when i get home.

Reply:
I'll work on it further later. At this point, it should work for now. I started messing with divs and getting it to look right, but I'm struggling. If you'd like to advise further guidance, I am open to it. But for now, I believe it should work.